sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -340,10 +346,38 @@ keyVaultAccessControlAsyncClient.deleteRoleAssignment(KeyVaultRoleScope.GLOBAL,
 ### Examples
 #### Sync API
 The following sections provide several code snippets covering some of the most common Azure Key Vault Backup client tasks, including:
+- [Pre-backup check for a Key Vault](#run-pre-backup-check-for-a-collection-of-keys)
 - [Backup a Key Vault](#backup-a-collection-of-keys)
+- [Pre-restore check for a Key Vault](#run-pre-restore-check-for-a-collection-of-keys)
 - [Restore a Key Vault](#restore-a-collection-of-keys)
 - [Restore a key](#selectively-restore-a-key)
 
+##### Run pre-backup check for a collection of keys
+Check if an entire collection of keys can be backed up by using `beginPreBackup()`.
+
+```java readme-sample-beginPreBackup
+String blobStorageUrl = "https://myaccount.blob.core.windows.net/myContainer";
+String sasToken = "sv=2020-02-10&ss=b&srt=o&sp=rwdlactfx&se=2021-06-17T07:13:07Z&st=2021-06-16T23:13:07Z&spr=https&sig=n5V6fnlkViEF9b7ij%2FttTHNwO2BdFIHKHppRxGAyJdc%3D";
```

Comment:
I know this is just a stand-in SAS, but I can imagine CredScan getting upset about this value. Is it possible to use a simpler, stand-in value like `"<SAS token>"`?

Reply:
Yeah, we can remove the fake SAS to avoid complaints from CredScan.